Sources/CircularProgress/util.swift

```diff
@@ -164,13 +164,13 @@ extension NSFont {
 
 
 extension NSBezierPath {
-	static func circle(radius: Double, center: CGPoint) -> NSBezierPath {
+	static func circle(radius: Double, center: CGPoint, startAngle: CGFloat = 0.0, endAngle: CGFloat = 360.0) -> NSBezierPath {
```

Comment:
I would prefer to use `Double` here instead of `CGFloat`.

Reply:
I keep forgetting your preference, hah.